pulsar-client-admin-shaded/pom.xml

```diff
@@ -364,10 +364,6 @@
                   <pattern>org.apache.pulsar.checksum</pattern>
                   <shadedPattern>org.apache.pulsar.shade.org.apache.pulsar.checksum</shadedPattern>
                 </relocation>
-                <relocation>
-                  <pattern>org.apache.pulsar.policies</pattern>
-                  <shadedPattern>org.apache.pulsar.shade.org.apache.pulsar.policies</shadedPattern>
-                </relocation>
```

Comment:
this change could be removed with the problem is correctly addressed in LoadReportDeserializer

Reply:
Instead of removing the relocation rule, add a exclude rule so that the classes and interfaces in https://github.com/apache/pulsar/tree/master/pulsar-client-admin-api/src/main/java/org/apache/pulsar/policies/data/loadbalancer aren't shaded:
```xml
  <excludes>
    <exclude>org\.apache\.pulsar\.policies\.data\.loadbalancer\.(LoadManagerReport|NamespaceBundleStats|ResourceUsage|ServiceLookupData)($.*)?</exclude>
  </excludes>
```

Reply:
Done.